tests/lit-tests/bool-store-varying.ispc

```diff
@@ -0,0 +1,25 @@
+// RUN: %{ispc} --target=host --nowrap --nostdlib -O2 --emit-llvm-text %s -o - | FileCheck %s
```

Comment:
`bool` type is tricky - it's representation differs on different platforms. Check how this works on AVX512 targets, this test may be failing.

Reply:
I have added explicit AVX512 runs.

Reply:
Now the test `REQUIRES: X86_ENABLED`